tests/foreman/ui/test_discoveryrule.py

```diff
@@ -61,8 +66,8 @@ def setUpClass(cls):
         cls.saved_per_page = str(cls.per_page.value)
         cls.per_page.value = '100000'
         cls.per_page.update({'value'})
-
-        cls.host_group = entities.HostGroup().create()
+        cls.host_group = entities.HostGroup(
+            organization=[cls.session_org]).create()
```

Comment:
I'd better add it. It sounds like a bug for me that location has no effect and sooner or later it may be fixed, and then your tests will fail. imo better to write it correct way so we won't receive any surprise failures later :)